front50-gcs/src/main/java/com/netflix/spinnaker/front50/config/GcsConfig.java

```diff
@@ -41,40 +41,44 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
 import java.util.Collections;
 
 @Configuration
+@ConditionalOnExpression("${spinnaker.gcs.enabled:false}")
 public class GcsConfig {
   // Refresh every 10 minutes. In practice this either doesnt matter because refreshes are fast enough,
   // or should be finer tuned. But it seems silly to refresh at a fast rate when changes are generally infrequent.
   // Actual queries always check to see if the cache is out of date anyway. So this is mostly for the benefit of
   // keeping other replicas up to date so that last-minute updates have fewer changes in them.
-  private static int APPLICATION_REFRESH_INTERVAL = 10 * 60;
-  private static int PROJECT_REFRESH_INTERVAL = 10 * 60;
-  private static int NOTIFICATION_REFRESH_INTERVAL = 10 * 60;
-  private static int PIPELINE_REFRESH_INTERVAL = 10 * 60;
-  private static int PIPELINE_STRATEGY_REFRESH_INTERVAL = 10 * 60;
+  private final Logger log = LoggerFactory.getLogger(getClass());
+  private static int APPLICATION_REFRESH_MS = (int)TimeUnit.MINUTES.toMillis(10);
+  private static int PROJECT_REFRESH_MS = (int)TimeUnit.MINUTES.toMillis(10);
```

Comment:
The problem is that the use site is an int. I thought it made more sense to cast here both because it is simpler and clear the magnitude fits.